std/typecons.d

```diff
@@ -6599,9 +6602,9 @@ Postcondition: `refCountedStore.isInitialized`
     }
 
     /// Ditto
-    this(T val)
+    this(return scope T val)
     {
-        _refCounted.move(val);
+        () @trusted { _refCounted.move(val); }();
```

Comment:
What is the purpose of this `@trusted`? Shouldn't `move` already infer safety, by taking e.g. `return ref` instead of `ref`? (BTW, **really** poor choice of method name there, although that's a flaw of original implementation and not this patch).

Reply:
You're right, it was needless. Removed.